.changes/unreleased/Under the Hood-20240711-105519.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
```

Comment:
Is this a breaking change if someone (other than dbt Core) is depending on DSI?

Reply:
MF is depending on the change but will be forced to change the references when upgrading DSI, so that's fine. MFS relies on DSI through MF. Anywhere else you're thinking of ?

Reply:
I don't know, does anybody use OSS DSI other than us?

Reply:
I think it should be fine regardless - the main concern we had was breaking people's manifests. Any other server should be fine to upgrade and migrate accordingly!